data_summaries/KIWI.json

```diff
@@ -0,0 +1,54 @@
+{
+    "KIWI": {
+        "Unique Dataset Identifier": "KIWI",
+        "Dataset Name": "KIWI",
+        "Paper Title": "KIWI: A Dataset of Knowledge-Intensive Writing Instructions for Answering Research Questions",
+        "Dataset URL": "https://www.cs.utexas.edu/~fxu/kiwi/",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/fangyuan/kiwi",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2403.03866",
+        "Semantic Scholar Corpus ID": "",
+        "Collection": "KIWI",
+        "Collection URL": "https://www.cs.utexas.edu/~fxu/kiwi/",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Question Answering",
+            "Scientific Knowledge Retrieval",
+            "Question Understanding",
+            "Open Question Answering",
+            "Academic Tasks"
+        ],
+        "Text Sources": [
+            "User interactions",
+            "Generated text"
+        ],
+        "Model Generated": [
+            "OpenAI GPT-3.5-Turbo",
+            "OpenAI GPT-4",
+            "LLaMA-2-chat-70b"
+        ],
+        "Format": [
+            "Multi-turn Dialog"
+        ],
+        "Human Annotation": "Yes",
+        "Derived from Datasets": [],
+        "Creators": [
+            "The University of Texas at Austin",
+            "University of Washington"
+        ],
+        "Licenses": [
+            {
+                "License": "CC BY-SA 4.0",
+                "License URL": "https://creativecommons.org/licenses/by-sa/4.0/legalcode"
+            }
+        ],
```

Comment:
Add OpenAI and Llama 2 license to this list.

Reply:
Added openai need help for llama.